src/code42cli/output_formats.py

```diff
@@ -88,10 +88,24 @@ def __init__(self, output_format):
             self._format_func = DataFrame.to_csv
         elif output_format == OutputFormat.RAW:
             self._format_func = DataFrame.to_json
-            self._output_args.update({"orient": "records", "lines": False})
+            self._output_args.update(
+                {
+                    "orient": "records",
+                    "lines": False,
+                    "index": True,
+                    "default_handler": str,
```

Comment:
Without this, we were getting an error when trying to convert the backup set info to json, I think because it was trying to traverse all the dynamic attributes of the `DeviceSettings` object in some weird way and hit the recursion limit.